app/ai/voice/agents/breeze_buddy/services/conversation_analysis/queue.py

```
@@ -0,0 +1,27 @@
+from app.core.logger import logger
+from app.services.redis import get_redis_service
+
+TOPIC_ANALYSIS_QUEUE = "conversation-analysis:pending"
+
+
+async def enqueue_topic_analysis(analysis_id: str) -> None:
+    try:
+        redis = await get_redis_service()
+        client: Any = cast(Any, await redis.get_client())
+        await client.rpush(TOPIC_ANALYSIS_QUEUE, analysis_id)
+    except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)